aidb/query/query.py

```diff
@@ -523,4 +574,34 @@ def add_where_condition(self, expression, operator:str , where_condition):
   def add_select(self, expression, selects):
     re = Rewriter(expression)
     e = re.add_selects(selects)
-    return e.expression
\ No newline at end of file
+    return e.expression
+
+
+  def add_join(self, expression, new_join):
```

Comment:
Isn't there a function that does something like this already?

Reply:
We have add_select, add_where but no add_join